posts/2022-07-02-sunny-tech-en.adoc

```diff
@@ -0,0 +1,227 @@
+= Sunny Tech 2022
+VincentBrule
+v1.0, 2022-07-02
+:title: Sunny Tech 2022
+:imagesdir: ../media/2022-07-02-sunny-tech
+:lang: en
+:tags: [conference, sunny tech, montpellier]
+
+== Sunny Tech ?
+https://sunny-tech.io/[Sunny Tech] took place from June 30 to July 1, under the sun of Montpellier.
+
+This conference covers a multitude of areas:
+
+. Cloud & Devops
+. Methodologies
+. Backend
+. Data
+. Web & Mobile & UI
+. Architecture
+. Atypical
+
+The organizers have prepared a _great schedule for us_.
+Thanks to the diversity of the topics presented, there was really something for every taste and profile.
+If you are interested, https://sunny-tech.io/schedule/[the program is available on the site].
```

Comment:
Feels a little like switching to present tense here - perhaps...

``suggestion
If you are interested, https://sunny-tech.io/schedule/[the program is still available on the site].
```